recipes/medical_imaging.ipynb

```diff
@@ -1,112 +1,181 @@
 {
```

Comment:
<p>In this example, <code>annotations</code> is a list containing 10 masks.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>